menpo/image/base.py

```diff
@@ -2525,17 +2530,38 @@ def pixels_range(self):
 
     def rolled_channels(self):
         r"""
+        Deprecated - please use the equivalent ``as_rolled_channels`` method.
+        """
+        warn('This method is no longer supported and will be removed in a '
+             'future version of Menpo. '
+             'Use .as_rolled_channels() instead.',
+             MenpoDeprecationWarning)
+        return self.as_rolled_channels()
+
+    def as_rolled_channels(self, out_dtype=None):
```

Comment:
completely support deprecating `as_imageio` but not sure `as_rolled_channels()` is the right name here, and if we are deprecating, might be worth a little discussion on getting it right.

For my money, that name to me doesn't suggest I'm going to get a numpy array back, so I'd assume an `Image` is the return type. That then makes me think that maybe menpo images support channels either rolled or not, which is not the case.

My other preference would be to move from `channels_rolled` generally to `channels_at_front` or `channels_at_back` respectfully. Rolling is the procedure to change from one to another, but it isn't clear to the callee what the input and output should be.

Those two thoughts lead me to suggest a name more like `.pixels_with_channels_at_back()` or even `.pixels_array_with_channels_at_back()`. That's obviously super verbose though, can we try and find something shorter than that that still conveys:

1. You are getting back a numpy array only
2. The channels on the array are at the back so you can use this array with code that expects that convention
3. Re-enfoces that menpo images must therefore always have channels not at the back, but the front.


Thoughts @patricksnape ?